### ISSUE TITLE
The extension opens fontforge at startup on linux

### ISSUE THREAD

**Issue by glepretre** [nicolo-ribaudo/brackets-svg-font#4]
![selection_002](https://cloud.githubusercontent.com/assets/5930831/7343942/9f5e23a8-ecc7-11e4-983f-17a56676850d.png)

**Comment by nicolo-ribaudo** [comment]
Can't reproduce on Ubuntu 14.10

Is FontForge opened by running `fontforge -v` in the terminal?

**Comment by glepretre** [comment]
> Can't reproduce on Ubuntu 14.10

I'm running Linux Mint Debian Edition :wink:

> Is FontForge opened by running fontforge -v in the terminal?

Yes.

My fontforge version is: `20120731.b-4`

I guess in that version, the command-line argument to get the system installed version is `-version`

```
fontforge --help
Copyright (c) 2000-2012 by George Williams.
 Executable based on sources from 14:57 GMT 31-Jul-2012-ML.
 Library based on sources from 14:57 GMT 31-Jul-2012.
fontforge [options] [fontfiles]
    -new             (creates a new font)
    -last            (loads the last sfd file closed)
    -recover none|auto|inquire|clean (control error recovery)
    -allglyphs       (load all glyphs in the 'glyf' table
             of a truetype collection)
    -nosplash        (no splash screen)
    -unique          (if a fontforge is already running open
             all arguments in it and have this process exit)
    -display display-name    (sets the X display)
    -depth val       (sets the display depth if possible)
    -vc val          (sets the visual class if possible)
    -cmap current|copy|private   (sets the type of colormap)
    -dontopenxdevices    (in case that fails)
    -sync            (syncs the display, debugging)
    -keyboard ibm|mac|sun|ppc  (generates appropriate hotkeys in menus)
    -usecairo=yes|no  Use (or not) the cairo library for drawing
    -usepango=yes|no  Use (or not) the pango library for text
    -usage           (displays this message, and exits)
    -help            (displays this message, invokes a browser)
                  (Using the BROWSER environment variable)
    -version         (prints the version of fontforge and exits)
    -library-status  (prints information about optional libraries
                 and exits)
    -lang=py         use python for scripts (may precede -script)
    -lang=ff         use fontforge's legacy scripting language
    -script scriptfile   (executes scriptfile)
        must be the first option (or follow -lang).
        All others passed to scriptfile.
    -dry scriptfile      (syntax checks scriptfile)
        must be the first option. All others passed to scriptfile.
        Only for fontforge's own scripting language, not python.
    -c script-string     (executes argument as scripting cmds)
        must be the first option. All others passed to the script.
```

Which version are you using? 
Because it seems I'm on the latest stable: [Debian packages search="fontforge"](https://packages.debian.org/search?keywords=fontforge&searchon=names&suite=all&section=all)

**Comment by fedor1113** [comment]
Hi!
I've had the same problem (Ubuntu 14.10). I opened the extensions folder and then found the .../node/fontforge/parser.ff file and changed _"fontforge -v"_ to _"fontforge -version"_. Everything works for now (after this hotfix the app stopped its opening with Brackets).

**Comment by nicolo-ribaudo** [comment]
Thanks, I'll release the patch in few hours.

Il giorno lun 27 apr 2015 17:13 fedor1113 notifications@github.com ha
scritto:

**Comment by fedor1113** [comment]
As a matter of fact, I've probably found a small lexical mistake in README.md. Here: "You don't have to do nothing!" (line 11), when it would be more correct to say "You don't have to do anything!"  =) You could add it to the patch as well.

**Comment by nicolo-ribaudo** [comment]
Released version 0.4.3
